frr_exporter.go

```diff
@@ -77,15 +58,15 @@ func main() {
 		}
 		landingPage, err := web.NewLandingPage(landingConfig)
 		if err != nil {
-			logger.Error(err.Error())
+			logger.Error("Couldn't create landing page", "err", err.Error())
```

Comment:
Negated status messages generally avoid the use of apostrophe contractions.
```suggestion
			logger.Error("Could not create landing page", "err", err.Error())
```